.travis.yml

```diff
@@ -0,0 +1,23 @@
+sudo: required
+
+services:
+  - docker
+
+env:
+  DOCKER_COMPOSE_VERSION: 1.18.0
```

Comment:
Does it matter that this is `1.18.0` but the docker-compose file itself says it's version 3?